scripts/test-upgrade.sh

```diff
@@ -22,30 +22,54 @@ BASE_DIR=$( cd "$( dirname ${0} )" && cd ../ && pwd )
 OA_DIR="$BASE_DIR/openstack-ansible"
 RPCD_DIR="$BASE_DIR/rpcd"
 
+# Create a backup of /etc/openstack_deploy, if one doesn't already exist
+# NOTE: This gets done by openstack-ansible's scripts/run-upgrade.sh script,
+#       but if we wait until then it'll be too late.
+if [[ ! -d "/etc/openstack_deploy.MITAKA" ]]; then
+  cp -a /etc/openstack_deploy /etc/openstack_deploy.MITAKA
```

Comment:
I agree with @sigmavirus24 it makes more sense (to me at least) for it to be openstack_deploy.LIBERTY - since that would be the version for which the variables are setup, its a simple change too.